displays/cavity_display/backend/backend_cavity.py

```diff
@@ -79,6 +81,9 @@ def create_faults(self):
 
             elif level == "SSA":
                 pv: str = self.ssa.pv_addr(suffix)
+                button_command = button_command.format(
+                    cm_OR_hl="hl" if self.cryomodule.is_harmonic_linearizer else "cm"
```

Comment:
What happens for the other SSA level faults who don't have the `cm_OR_hl` format in their button path?

Reply:
Nothing! This was part of the output when I printed it

> $EDM/llrf/rf_srf_ssa_cm.edl
> $PYDM/nirp/nirp_Lband_diagnostic_graphical.ui
> 
> $EDM/llrf/rf_srf_ssa_hl.edl
> $PYDM/nirp/nirp_Lband_diagnostic_graphical.ui

Reply:
Oh nice! Okay, cool:D Just wanted to make sure it didn't mess with the other SSA faults